tox.ini

```diff
@@ -251,7 +251,7 @@ commands_pre =
 
   aiopg: pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-dbapi pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-aiopg[test]
 
-  datadog: pip install {toxinidir}/exporter/opentelemetry-exporter-datadog
+  datadog: pip install flaky {toxinidir}/exporter/opentelemetry-exporter-datadog
```

Comment:
Not sure if this is the right place to put this dependency, but after trying in several places, this is the one that worked. Any suggestions @NathanielRN?

Reply:
this works fine. generally it's nice to put it into a multi-name match above this one:

```
datadog,other-instrumentation-that-needs-it: pip install flaky
datadog: {toxinidir}/..../exporter-datadog
```

Reply:
Tthis is the right place to put it (since only `datadog` needs it for now) but I would also like to know why the test sometimes fail 😕